src/views/ModelPlan/ReadOnly/index.tsx

```diff
@@ -184,47 +190,65 @@ const ReadOnly = () => {
   const subComponents: subComponentsProps = {
     'model-basics': {
       route: `/models/${modelID}/read-only/model-basics`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/model-basics',
       component: <ReadOnlyModelBasics modelID={modelID} />
     },
     'general-characteristics': {
       route: `/models/${modelID}/read-only/general-characteristics`,
+      helpRoute:
+        '/help-and-knowledge/sample-model-plan/general-characteristics',
       component: <ReadOnlyGeneralCharacteristics modelID={modelID} />
     },
     'participants-and-providers': {
       route: `/models/${modelID}/read-only/participants-and-providers`,
+      helpRoute:
+        '/help-and-knowledge/sample-model-plan/participants-and-providers',
       component: <ReadOnlyParticipantsAndProviders modelID={modelID} />
     },
     beneficiaries: {
       route: `/models/${modelID}/read-only/beneficiaries`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/beneficiaries',
       component: <ReadOnlyBeneficiaries modelID={modelID} />
     },
     'operations-evaluation-and-learning': {
       route: `/models/${modelID}/read-only/operations-evaluation-and-learning`,
+      helpRoute:
+        '/help-and-knowledge/sample-model-plan/operations-evaluation-and-learning',
       component: <h1>operationsEvaluationAndLearning</h1>
     },
     payment: {
       route: `/models/${modelID}/read-only/payment`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/payment',
       component: <ReadOnlyPayments modelID={modelID} />
     },
     'it-tools': {
       route: `/models/${modelID}/read-only/it-tools`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/it-tools',
       component: <h1>itTools</h1>
     },
     team: {
       route: `/models/${modelID}/read-only/team`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/team',
       component: <ReadOnlyTeamInfo modelID={modelID} />
     },
     discussions: {
       route: `/models/${modelID}/read-only/discussions`,
+      helpRoute: '/help-and-knowledge/sample-model-plan/discussions',
       component: <ReadOnlyDiscussions modelID={modelID} />
     },
     documents: {
       route: `/models/${modelID}/read-only/documents`,
-      component: <ReadOnlyDocuments modelID={modelID} />
+      helpRoute: '/help-and-knowledge/sample-model-plan/documents',
+      component: (
+        <ReadOnlyDocuments modelID={modelID} isHelpArticle={isHelpArticle} />
+      )
     },
     'crs-and-tdl': {
       route: `/models/${modelID}/read-only/crs-and-tdl`,
-      component: <ReadOnlyCRTDLs modelID={modelID} />
+      helpRoute: '/help-and-knowledge/sample-model-plan/crs-and-tdl',
+      component: (
+        <ReadOnlyCRTDLs modelID={modelID} isHelpArticle={isHelpArticle} />
+      )
     }
   };
 
```

Comment:
I think here is where you should filter on `isHelpArticle` boolean.  I think `Discussions` should be removed completely as a component if `isHelpArticle` is true, otherwise I can type in `/help-and-knowledge/sample-model-plan/discussions` and still hit the component and make updates.  This may also simplify other areas of conditional rendering of `subComponent`

I think there should also be handling of sub routes that don't exist.  The app crashes on any route that isn't a configured route.  `/help-and-knowledge/sample-model-plan/discussions2` causes a crash.

It can possible be handled here:
```
if ((!loading && error) || (!loading && !data?.modelPlan)) {
    return <NotFoundPartial />;
  }
```